rtl_wmbus.c

```diff
@@ -766,12 +807,10 @@ int main(int argc, char *argv[])
 
         for (size_t k = 0; k < sizeof(samples)/sizeof(samples[0]); k += 2)   // +2 : i and q interleaved
         {
-            const float i_unfilt = ((float)samples[k]     - 127.5f);
-            const float q_unfilt = ((float)samples[k + 1] - 127.5f);
-
-            // rtl_sdr -f 868.35M -s 2400000 - 2>/dev/null | build/rtl_wmbus -d 3
-            //shift_freq(&i_unfilt, &q_unfilt, 600, 2400);
+            const float i_unfilt = ((float)samples[k]     - 127.5);
+            const float q_unfilt = ((float)samples[k + 1] - 127.5);
```

Comment:
would be double operation. samples[k] - 127.5f should be left here.

Reply:
Had an outdated version without this